build/tasks/AddMetapackageReferences.cs

```diff
@@ -81,7 +101,12 @@ public override bool Execute()
             {
                 var packageName = package.ItemSpec;
                 var packageVersion = package.GetMetadata("Version");
-                var packageVersionValue = LockToExactVersions ? $"[{packageVersion}]" : packageVersion;
+                var packageVersionValue = GetDependencyVersion(dependencyVersionType, packageName, packageVersion);
+                if (packageVersionValue == null)
+                {
+                    Log.LogMessage("Skipping {0}", packageName);
```

Comment:
When does this happen? This would probably be an error.

Reply:
Should never happen. I'll remove it.

Reply:
Throw an exception since we would probably like to know when this happens as it would generate [1.0.0, ) which would be wrong.

Reply:
Updated.